webapp/src/js/components/Map/TableMarkersLayer/Widget.js

```diff
@@ -139,7 +132,7 @@ let TableMarkersLayer = React.createClass({
           columns: locationColumns,
           database: this.config.dataset,
           query: this.getDefinedQuery(query),
```

Comment:
Doesn't this call use this.props.table? If the table has changed then this will incorrectly use the old table.